src/hooks/useAllowance.ts

```diff
@@ -1,4 +1,4 @@
-import { useMemo } from "react";
+import { useCallback, useEffect, useMemo, useState } from "react";
```

Comment:
`useCallback` and `useMemo` are unused.

Also line 3, `import { WETH }` is unused